scannerlib/configchecks/group_criteria.go

```diff
@@ -30,6 +30,10 @@ var (
 	umaskRe = regexp.MustCompile("^0?[0-7][0-7][0-7]$")
 )
 
+var (
+	swVersRe = regexp.MustCompile("^\\d+.\\d+.\\d+-\\d+\\+.*$")
```

Comment:
Add to previous var() block
Also no need to use abbreviations - let's call it softwareVersionRe